pyproject.toml

```diff
@@ -95,9 +94,7 @@ dev = [
     # PyTorch
     "torch>=1.12.0,<3.0.0",
     "torchvision>=0.13.0",
-    # TODO: remove the constraint once tf2onnx supports onnx>=1.15.0
-    # TODO: cf. https://github.com/onnx/tensorflow-onnx/pull/2252
-    "onnx>=1.12.0,!=1.15.0,<3.0.0",
+    "onnx>=1.12.0,<3.0.0",
```

Comment:
So, should we enforce `1.16.0` instead ?

Reply:
Mh no that's more tf2onnx related we could set the lower bound to latest !?

Reply:
i have increased the lower tf2onnx version

Reply:
Yaeh sorry, I pointed out the wrong part of the code. I was thinking about `tf2onnx`